C++ Programs/Stacks/postfix_evaluation.cpp

```diff
@@ -0,0 +1,64 @@
+#include <bits/stdc++.h>
+
+#define OPERAND 1
+#define OPERATOR 0
+
+// to check if the current character is an operand
+int isOperand(char oper);
+
+using namespace std;
+
+// DRIVER FUNCTION
+int main()
+{
+    // Input is the postfix expression
+    char postfix[] = "73426^*+*8+";
+    int res, postfix_ind, postfix_size, op1, op2;
```

Comment:
Writing clean code is a great part of programming, not for CP contests, but for others to understand your code.

Some points to be kept in mind,

We can use proper variable names, like we can use `postfix_index` etc.
We can use better names for `op1` & `op2`.